### PR TITLE
Use brightlayer packages for MUI demo

### PR DESCRIPTION
**Pull request by joebochill**
I had to do this to test something out, so figured I'd push it up.

#### Changes proposed in this Pull Request:
- Switch mui showcase to use brightlayer pacakges
- Update mui showcase to use grid to match layout of the icon-font demo file

#### To Test:
- yarn start

### CODE REVIEW

File: mui/demos/showcase/src/index.tsx [hunk 1]
```diff
@@ -3,8 +3,8 @@ import 'react-app-polyfill/stable';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { CssBaseline, ThemeProvider, createMuiTheme } from '@material-ui/core';
-import * as PXBThemes from '@pxblue/react-themes';
-import '@pxblue/react-themes/open-sans';
+import * as PXBThemes from '@brightlayer-ui/react-themes';
```
Comment (emclaug2) [inline mui/demos/showcase/src/index.tsx:6]: import * as BLUIThemes from '@brightlayer-ui/react-themes';

**Comment by emclaug2** [comment]
Looks good otherwise!

**Review by emclaug2 — APPROVED** [review]

**Comment by bkarambe** [comment]
@joebochill There is a pxblue reference in Changelog.md at line 47. 

- [ ] Remove pxblue reference in Changelog

**Comment by bkarambe** [comment]
@joebochill Missed pxblue reference replacement in mui/Changelog.md at line 116.

**Comment by bkarambe** [comment]
```tsx
// new import syntax
import IconName from '@pxblue/icons-mui/IconName';
```

- [ ] Please replace pxblue to brightlayer-ui in Changelog.md

**Comment by bkarambe** [comment]
Please refactor Logo.tsx to remove px blue reference

- [ ] Remove pxb references from Logo.tsx

**Review by bkarambe — CHANGES_REQUESTED** [review]
Please address the feedback

**Comment by bkarambe** [comment]
> I had to do this to test something out, so figured I'd push it up.
> 
> #### Changes proposed in this Pull Request:
> * Switch mui showcase to use brightlayer pacakges
> * Update mui showcase to use grid to match layout of the icon-font demo file
> 
> #### To Test:
> * yarn start

Please mention jira ticket number

**Comment by joebochill** [comment]
Thanks for the feedback @bkarambe. Just to be clear, this PR is _not_ tied to a story. This is simply me fixing a bug with the demo so that it shows the correct icons. If there are other BLUI migration issues with the MUI demo, that story is currently assigned to you. If there are lingering issues with the BLUI migration for the icons packages (PXBLUE-2523, which was closed last sprint), then you can either address them as part of your story, or make a new issue in JIRA to come back and clean them up.

And for what it's worth...I don't think the Logo.tsx file is used at all in the demo (it can probably be removed). The mui changelog is fine because that line explicitly refers to the syntax for the old version of the package (the brightlayer scoped package does not exist at that version). And the /changelog file is pointing to a bug in our github org, which hasn't been renamed yet. @emclaug2 not sure if we'll want to handle all of these as a post-zero-day update (if so we should make sure that it's part of the AC for a story), or if we should update them ahead of time and let them be broken until zero day. Your call.

**Comment by bkarambe** [comment]
@joebochill Thanks for the update. Approving PR everything goods.

**Review by bkarambe — APPROVED** [review]

**Comment by jeffvg** [comment]
should updated sprites be checked in? I ran build:font and it includes sprites but didn't run. If you run sprites alone we have updated sprites

**Comment by emclaug2** [comment]
And the /changelog file is pointing to a bug in our github org, which hasn't been renamed yet. @emclaug2 not sure if we'll want to handle all of these as a post-zero-day update (if so we should make sure that it's part of the AC for a story), or if we should update them ahead of time and let them be broken until zero day. Your call.


I'd rather update the links now; @bkarambe could you take a look?

**Comment by bkarambe** [comment]
@emclaug2 Sure. I will cover it as a part of PXBLUE-2609.

**Pull request merged**